SECURITY.md

```diff
@@ -0,0 +1,9 @@
+# Security Policy
+
+This is a static website which does not have security policies.
+This file is here to prevent GitHub from mistaking another file as security policy.
+
+## Reporting a Vulnerability
+
+Open an Issue or a PR.
```

Comment:
Honestly this is a bit naive and makes us look very amateurish. Sure, it's a static website, but we also provide code snippets and, though unlikely, there may be security issues or vulnerabilities in those snippets. Furthermore, saying "open an issue" completely defeats the purpose of a dedicated process for vulnerability reporting (which should be private until the vuln is fixed) and goes against all best practices of responsible disclosure.

If GitHub showing a random unrelated file as our "security policy" is a problem, I would suggest just renaming that file to prevent GitHub from treating it specially.

Reply:
Make sense. Let's rename the file.